api/handler/put_test.go

```diff
@@ -86,3 +89,18 @@ func TestCustomJSONMarshal(t *testing.T) {
 
 	require.Equal(t, expectedPolicy, policy)
 }
+
```

Comment:
Should we also replace `policy := &postPolicy{}` to `policy :=  &postPolicy{empty: true}` in line 86, even though it doesn't affect the test?

Reply:
I'm not sure because not empty policy is tested there.